scripts/save_pt_dummy_model.py

```diff
@@ -0,0 +1,21 @@
+import torch.nn as nn
```

Comment:
Thanks! This script is very handy to test things. Small point but I don't think torch is added to the Poetry environment yet.